data/korath/korath ships.txt

```diff
@@ -329,17 +330,17 @@ ship "Korath Ravager"
 		angle 190
 	gun 98 75 "Firelight Missile Bank"
 		angle 170
-	turret -32.5 -53.5 "Korath Inferno"
-	turret 32.5 -53.5 "Korath Inferno"
+	turret -32.5 -53.5 "Langrage Hyper-Heaver"
+	turret 32.5 -53.5 "Langrage Hyper-Heaver"
 	turret 0 -23 "Korath Warder"
 	explode "tiny explosion" 120
 	explode "small explosion" 60
 	explode "medium explosion" 70
 	explode "large explosion" 50
 	explode "huge explosion" 15
 	"final explode" "final explosion large"
-	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed wholesale at the largest Korath facilities, something that, until recently, was generally reserved for the likes of the massive Ik'atilajej and the Korsmanath A'awoj."
-	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller Fire-Lances allowing to burn enemies in close quarters."
+	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly unique warship design the Exiles have fielded since the Exodus, but also the first built solely for combat instead of needing to double as a habitat or supply ship. Its size and non-modular nature also means it must be constructed wholesale at the largest Korath facilities, something that, until recently, was generally reserved for the likes of the massive Ik'atilajej and the Korsmanath A'awoj."
+	description "	The 'nra'ret-Class Chaser, named in honor of a brilliant weapons engineer who took their own life to avoid having their secrets captured by a hostile alien force, provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units. To this end, it is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of langrage heavers for sustained operations, and two Fire-Lances for emergency use in close quarters."
```

Comment:
I assume the `langrage heavers` doesn't use the exact outfit name on purpose.

Reply:
It seemed unnecessary to me to be that specific, "langrage heavers" flows more naturally into something one would actually say vs. the technical title Langrage Hyper-Heavers.